src/components/Slider/Slider.js

```diff
@@ -76,6 +78,8 @@ export default function Slider({
 }
 
 Slider.propTypes = {
+  /** Index of first slide */
```

Comment:
This could be a bit confusing.

Perhaps change to something like -> `Index of the slide to be rendered by default`?

Reply:
Done